bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_reweight_card.dat

```diff
@@ -0,0 +1,500 @@
+ change rwgt_dir ./rwgt
+ change mode LO  
+ change helicity False 
+
+ launch  
```

Comment:
why not adding names here? will be useful to udnerstand.

Reply:
We just cloned this file from the files used for the published 2016 analysis: https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX/WWTolnulnu_01j_aTGC_lep_WWmass-0to400_4f_NLO_FXFX/WWTolnulnu_01j_aTGC_lep_WWmass-0to400_4f_NLO_FXFX_reweight_card.dat